### PR TITLE
Fill map with fixture data

### PR DESCRIPTION
**Pull request by jciepka**
image

### CODE REVIEW

File: src/others/components/Map.tsx [hunk 1]
```diff
@@ -15,6 +54,12 @@ export function Map() {
   const [lat] = useState(ukraineCenterCoords.lat);
   const [zoom] = useState(4);
```
Comment (jschlenker) [inline src/others/components/Map.tsx:55]: *non blocking*
I don't believe `lng`, `lat` nor `zoom` need to be stateful.
Defining
```typescript
const initialView= {
    lat: 48.4501071,
    lng: 30.5240501,
    zoom: 4,
  };
```
and passing it directly to `initialViewState` should do the job.

File: src/others/components/Map.tsx [hunk 2]
```diff
@@ -1,9 +1,48 @@
...
-import MapComponent from "react-map-gl";
+import MapComponent, { Layer, LayerProps, Source } from "react-map-gl";
 import maplibregl from "maplibre-gl";
 import "maplibre-gl/dist/maplibre-gl.css";
 
 import styles from "./Map.module.css";
+import { aidRequestsFixture } from "../fixtures/request.fixture";
+import { FeatureCollection, GeoJsonProperties, Geometry } from "geojson";
+
+const layerStyle: LayerProps = {
```
Comment (jschlenker) [inline src/others/components/Map.tsx:10]: *non blocking*

Should probably live in a separate file, especially when additional layers come in.

Comment (jschlenker) [inline src/others/components/Map.tsx:10]: *non blocking*

Should probably live in a separate file, especially when additional layers come in.

File: src/others/fixtures/request.fixture.tsx [hunk 3]
```diff
@@ -0,0 +1,3881 @@
+export const aidRequestsFixture = [
```
Comment (jschlenker) [inline src/others/fixtures/request.fixture.tsx:1]: *non blocking*
```suggestion
import type { Feature, Geometry, GeoJsonProperties } from "geojson";

export const aidRequestsFixture: Feature<Geometry, GeoJsonProperties>[] = [
```

Comment (jschlenker) [inline src/others/fixtures/request.fixture.tsx:1]: *non blocking*
```suggestion
import type { Feature, Geometry, GeoJsonProperties } from "geojson";

export const aidRequestsFixture: Feature<Geometry, GeoJsonProperties>[] = [
```

File: src/others/components/Map.tsx [hunk 4]
```diff
@@ -15,6 +54,12 @@ export function Map() {
   const [lat] = useState(ukraineCenterCoords.lat);
   const [zoom] = useState(4);
 
+  const geojson: FeatureCollection<Geometry, GeoJsonProperties> = {
+    type: "FeatureCollection",
+    //@ts-ignore
```
Comment (jschlenker) [inline src/others/components/Map.tsx:59]: *non blocking*
TypeScript is your friend 😉 See `src/others/fixtures/request.fixture.tsx`
```suggestion
```

Comment (jschlenker) [inline src/others/components/Map.tsx:59]: *non blocking*
TypeScript is your friend 😉 See `src/others/fixtures/request.fixture.tsx`
```suggestion
```

**Review by jschlenker — COMMENTED** [review]
Seems like a great start. But I'm just snooping from S3 so no approval from my side at this point 😄

File: src/others/components/Map.tsx (hunk 2 above)
Comment (jciepka) [reply src/others/components/Map.tsx:10]: Totally agree!

File: src/others/components/Map.tsx (hunk 1 above)
Comment (jciepka) [reply src/others/components/Map.tsx:55]: Done

File: src/others/fixtures/request.fixture.tsx (hunk 3 above)
Comment (jciepka) [reply src/others/fixtures/request.fixture.tsx:1]: Done

File: src/others/components/Map.tsx (hunk 4 above)
Comment (jciepka) [reply src/others/components/Map.tsx:59]: Great!

**Comment by jciepka** [comment]
Thanks so much @jschlenker! My 2 years break from TypeScript and no experience with React came out.. But doing my best to at least add something to project ;)

**Comment by wtfua92** [comment]
Could we make `Map` accept `Source` and `Layer` as children components? in this way we can reuse the wrapping component for both requests and incidents

**Review by AntonRaichuk — APPROVED** [review]

**Review by MaciejReimann — APPROVED** [review]

**Pull request merged**